pyproject.toml

```diff
@@ -62,6 +62,8 @@ mypy = "^0.910"
 types-PyYAML = "^5.4.6"
 types-requests = "^2.25.6"
 scriv = {extras = ["toml"], version = "^0.12.0"}
+tox = "^3.24.4"
+pre-commit = "^2.15.0"
```

Comment:
Should we add these in the follow up PR (I think you mentioned the next one would be adding tests?)

Reply:
It's possible. BUT! I changed these lines in the same commit like a genius and didn't want to rip apart the commit at that level (and regenerate `poetry.lock`) to remove these. As they're dev dependencies that have no config files triggering them yet I would like to leave them in here. Let me know if that's acceptable in this situation.